pootle/apps/pootle_format/default.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+
+POOTLE_FORMATS = [
+    ("po",
+     dict(title='Gettext PO file',
+          extension="po",
+          template_extension="pot")),
+    ("xliff",
+     dict(title='XLIFF Translation File',
+          extension="xliff",
+          template_extension="xliff")),
+    ("xlf",
+     dict(title='XLIFF Translation File',
```

Comment:
I am not seeing anything like that in https://github.com/translate/pootle/pull/4966/commits/59d58aa7d11eafa6a4dcf84994ea562def85b738#diff-99489b5e96e0f37f2daaa11c328f4d0dR31